config/initializers/braintree.rb

```diff
@@ -1 +1,12 @@
 Spree::Admin::PaymentsController.helper :braintree_admin
+
+if Spree::Backend::Config.respond_to?(:menu_items) && Rails.env != "test"
```

Comment:
I know you said that this was required for test, but I would expect to see a fully reciprocal conditional in the other change block (here, in `admin_navigation_menu.rb`) that forces the Deface code to run if you _are_ in test. Why is that not needed?

Reply:
Good point. The issue is that the routes generated by the extension in some cases rely on another extension and as such they don't exists and calling the Railsified `admin_#{whatever}_path` method blows up the app.